Resources/AlarmTestsResources/ahuFailedToTurnOffAlarmResources.robot

```diff
@@ -0,0 +1,86 @@
+*** Settings ***
+Documentation          This resource file provides the keyword definition specific to Alram1-1 testsuite
+#...                    Created on 8th Dec 2021
+Library    RequestsLibrary
+Library    JSONLibrary
+Library    Collections
+Library     String
+Library    SeleniumLibrary
+Library    ${EXECDIR}/ExternalKeywords/common.py
+Variables    ${EXECDIR}/Configurations/${environment}.py
+Resource    ${EXECDIR}/Resources/apiresources.robot
+Resource    ${EXECDIR}/Resources/connection.robot
+Resource    ${EXECDIR}/Resources/common.robot
+Resource    ${EXECDIR}/Inputs/testInputs.robot
+Resource    ${EXECDIR}/Inputs/GraphQL/gqlQueries.robot
+
+
+*** Keywords ***
+ahuFailedToTurnOffAlarmTestPreconditionSetup
+    [Documentation]    Stop all VEM processes and wait for 2 minutes
+    ...                Also write test entry temperature for the parallel staleStatePrevention program
+    log to console    !-----Reading the inputs from the excel and storing in dictionary------!
+    testInputs.readingInputsFromExcel  alarmTest  A  B
+    common.setFlagValue    ${test_entry_flag}
+    log to console    !-----PreCondition for the Alarm-1 test is been executed------!
+    connection.establishConnectionAndStopAllProcessesExcept
+    common.waitForMinutes    2
+    apiresources.writeTestEntryTemperatureToSensorsAfterVXServerStarted
+
+setAllowNumExceedencesGuardCATGuardBandRangeOnPwrLvl
+    apiresources.changeCxConfigsTabModuleFieldValues  DASHM  AllowNumExceedencesGuard  ${test_input}[allow_num_exceedences_guard]
+    apiresources.changeCxConfigsTabModuleFieldValues  DASHM  CATGuardBandRange  ${test_input}[cat_guard_band_range]
+    ${ahuoid}=    apiresources.getOidOfComponentUsingComponentName   ${test_input}[ahu_change_onpwrlvl]
+    log to console     ${ahuoid}
+    apiresources.setComponentPropertyValue   ${ahuoid}     OnPwrLvl    float      ${test_input}[ahu_property_onpwrlvl]
+
+verifyNoAlarmAHUFailedToTurnOffRaised
```

Comment:
Not required if using apiresources.checkingAlarmStatusForGroup

Reply:
Done